src/Versatile_RotaryEncoder.h

```diff
@@ -13,19 +13,40 @@
 #ifndef VERSATILE_ROTARYENCODER_H_INCLUDED
 #define VERSATILE_ROTARYENCODER_H_INCLUDED
 
+/**
+ * Decomment below line to enable std::function as callbacks
+*/
+#define USE_STD_FUNCTION
+
+/**
+ * Decomment below line to disable double press and receive press event immediately
+*/
+// #define DISABLE_DOUBLE_PRESS
+
+/**
+ * Decomment below line to use internal pullup resistor
+*/
+#define USE_INTERNAL_PULLUP
+
+#if defined(USE_INTERNAL_PULLUP)
+#define INPUT_TYPE INPUT_PULLUP
+#else
+#define INPUT_TYPE INPUT
+#endif
+
 #include <Arduino.h>
 
 class Versatile_RotaryEncoder {
     public:
         enum Encoder : uint8_t {inactive, release, press, hold, rotate, pressrotate, heldrotate};
         enum Button : uint8_t {released, holdup, switchup, switchdown, pressed, holddown, held};
```

Comment:
I will need to add "double_press" state for Button enum instead of the "check_double_press" flag.